modules/ROOT/partials/configuration/color_picker_callback.adoc

```diff
@@ -3,7 +3,7 @@
 
 This option enables you to provide your own color picker.
 
-Type : `+JavaScript Function+`
+Type : `+Function+`
 
 === Example: Using `+color_picker_callback+`
```

Comment:
we didn't convert this one to a promise?

Reply:
It's not in the `Initial configuration` section... this only got changed as I did a Find and Replace on `JavaScript Function` in the type.

That said, `color_picker_callback` doesn't even exist anymore and this file isn't included anywhere in the docs, so we should delete this file.

Reply:
Deleted the unused page.